play.pokemonshowdown.com/src/panel-popups.tsx

```diff
@@ -300,6 +302,22 @@ class OptionsPanel extends PSRoomPanel {
 		}
 		PS.update();
 	};
+
+	handleShowStatusInput = (ev: Event) => {
+		ev.preventDefault();
+		ev.stopImmediatePropagation();
+		this.setState({ showStatusInput: !this.state.showStatusInput });
+	};
+
+	editStatus = (ev: Event) => {
+		const statusInput = this.base!.querySelector<HTMLInputElement>('input[name=statustext]');
+		if (!statusInput?.value.length) return;
+		PS.rooms['']?.send(`/status ${statusInput?.value || ''}`);
```

Comment:
```suggestion
		PS.send(`|/status ${statusInput?.value || ''}`);
```

(You could also do `PS.mainmenu.send` but I've been using this invocation basically everywhere else...)